human_clinical_and_health_data/data_sources.md

```diff
@@ -1,63 +1,142 @@
+* Health data repositories: in some countries, national or regional health data repositories have been established to aggregate electronic health records across multiple sources. For example, in Sweden, [the National Patient Register](https://www.socialstyrelsen.se/en/statistics-and-data/registers/national-patient-register/) collects data from all healthcare providers in the country; while in the UK, [the National Health Service (NHS) Digital](https://digital.nhs.uk/) collects data from various healthcare settings. However, in many countries these types of registries are not yet in place.
+* Health research databases: in a few countries, national health research databases have been established to support scientific research. These databases typically include de-identified data from large patient populations.
+* Data protection laws: all countries within the European Union are subject to the [General Data Protection Regulation (GDPR)](https://eur-lex.europa.eu/eli/reg/2016/679/oj), which establishes strict standards for data privacy and protection. European countries however have different interpretations of the GDPR and different national regulations on health and research data, adding complexity on how health data can be collected, stored, and shared. For example, across European member states, some countries have sectoral laws concerning the release of patient data for research purposes while others do not (e.g. Romania, Slovakia and Sweden). Countries in turn also have a different legal basis that they rely upon for the secondary use of health data for research by third-party public-sector researchers (e.g. Explicit Consent; Broad consent as defined in national legislation or no specific legislation). Also within a country the different definitions and interpretations can form a barrier. For example, in Germany there are Data Protection Authorities (DPAs) in each region as well as at a national level which have different definitions of the secondary use of health data. An in-depth overview of the country differences can be found in the report on [‘Assessment of the EU Member States’ rules on health data in the light of GDPR’](https://op.europa.eu/s/yLde).
+* Cross-border health data exchange: The European Union is working to establish a common framework for cross-border exchange of health data: [The European Health Data Space (EHDS)](https://health.ec.europa.eu/ehealth-digital-health-and-care/european-health-data-space_en). This initiative aims to facilitate the exchange of health data across borders while ensuring data protection and privacy. The EHDS will include a set of technical standards and protocols to enable interoperability of health data across Europe. Currently the European Genome-phenome Archive (EGA) is expanding into a federation of nodes through the [Federated EGA](https://ega-archive.github.io/FEGA-onboarding/). FEGA enables deposition and distribution of health data that cannot be stored outside of the country of origin.
```

Comment:
Here I think you can make FEGA and EHDS different bullet points. 

For the EHDS, you could add that one of the key things it aims to do is improve findability of data through metadata catalogues using metadata standards.